src/DIRAC/FrameworkSystem/scripts/dirac_login.py

```diff
@@ -37,6 +37,9 @@
     getTokenFileLocation,
 )
 
+DEFAULT_AUTH_WAY = "certificate"
+DEFAULT_RESPONSE = "proxy"
```

Comment:
What's possible (the strings "certificate", "proxy", "token" etc) should be defined in a separate module IMHO, as these strings IIUC will certainly be used elsewhere too.

Reply:
I added comments about possible variables

These values are tied to the keys used by the user, so I defined the default values here and this way. I think that if there will be a precedent for the wider use of these values, then they can be defined in another, more appropriate place

Reply:
I was more thinking of adding a file like https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/Core/Security/Properties.py

Now I merge it but please add it to another PR.